src/libcharon/plugins/kernel_super/kernel_super_ipsec.c

```diff
@@ -284,13 +409,12 @@ CALLBACK(terminator_ka_timer_cb, job_requeue_t,
 }
 
 static void count_term_tunnels_number_cb(void* ctx, const struct terminators_table_iterator* iterator) {
-	(void) ctx;
-	write_terminator_tunnels_oper(iterator->term_conn);
+	write_terminator_tunnels_oper((private_kernel_super_ipsec_t*)ctx, iterator->term_conn);
```

Comment:
If you expect the first parameter to be `private_kernel_super_ipsec_t*`, then use the `CALLBACK()` macro.